ceph.spec.in

```diff
@@ -14,6 +14,13 @@
 #
 # Please submit bugfixes or comments via http://tracker.ceph.com/
 #
+
+#################################################################################
+# conditional build section
```

Comment:
Nit: what's with this section header? It seems like something that is really trying to grab someone's attention for a critical concern. I think your comment update on line 61 effectively clears up the confusion re: the syntax.

Reply:
I like it. I think it puts the relevant info at your fingertips when wrestling with these conditional variables. Could've used it yesterday! :)